src/content/basics/giant-swarm-platform-access/index.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "The Giant Swarm Platform Access Routes"
+description = "General Giant Swarm Platform Access Routes Overview"
+date = "2019-05-08"
+weight = 20
+type = "page"
+categories = ["basics"]
++++
+
+# Giant Swarm Platform Access Routes
+
+## Intro
+Access to Giant Swarm platforms can be split into two parts. 
+
+1. Admin Access - designated for the Giant Swarm employees for managing/development/support purposes.
+
+2. User Access - designated for GiantSwarm customers to interact with the offered services.
+
+Both are managed via Giant Swarm VPN hopp that enables connection through jumphost of the cluster.
+Giant Swarm uses two different providers of VPN to improve the resilience of offered services.
+
+## General VPN Connection Schema
+
+![](/img/site-to-site-vpn.png)
+
+## Admin Access
+
+VPN access is managed via certificates (public/private keys) and secured connection between both VPNs.
+Certificates management is handled by one of GS private components. 
+The general principle is that certificate is configured for each individual Giant Swarm employee.
+
+VPN Secured access points:
+
+* SSH - SSH Admin access is based on the Github SSO. Only users residing in the Github GS Organization are allowed to authenticate.  
+  Following process describes more in detail the procedure of SSH authentication:
+  ![](/img/ssh_access_process.png)
+
+  SSH access to CP allows Giant Swarm also to manage and connect to underlying Data Platforms of the Customer.
```

Comment:
I would explain what is `CP`, so people not familiarized with our platform can understand it

Reply:
+1. As above, please use the long form when first introducing the acronym.